src/main/resources/application-test.properties

```diff
@@ -0,0 +1,11 @@
+spring.datasource.url=jdbc:oracle:thin:@localtracker_high?TNS_ADMIN=./wallet
```

Comment:
I have a question. Did you deleted the last tests that I've done?

Reply:
No Jhony, your tests are exactly how they were made.